plugin/slime.vim

```diff
@@ -52,22 +52,41 @@ endfunction
 " Tmux
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
+function! s:TmuxDefaultSocket()
```

Comment:
It could, but so can everything in the plugin. I think the purpose of a plugin is to capture common tasks and configurations and make them easily reusable. I think the 90% case is that a user wants to send keys to a pane in the same tmux session. This is that default. I'll remove it, but would it be fine if I moved it to the README as an example? At least then users can just copy-paste (which I believe is the second most common choice for initial configuration, after "no configuration, use the defaults").